src/renderer/src/components/common/Select.tsx

```diff
@@ -140,7 +140,16 @@ export const Select = ({ label, className = '', options = [], value, onChange, s
                 role="option"
               >
                 <div className="flex items-center">
-                  <span className="block truncate">{opt.label}</span>
+                  <span
+                    className="block truncate"
+                    style={{
```

Comment:
This should not be Select specific. If you want to have something that's `Option` specific, rather add `style?: CSSProperties` property to `Option`.